diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+GitHub is an online repository for code that allows multiple people to work on the same project and control the version of the code that is published
```

Comment:
Great! another answer could be 'A user interface (UI) for accessing git's version control capabilities for our projcet repositories.'